sdk/translation/azure-ai-translation-text/setup.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#--------------------------------------------------------------------------
+
+from io import open
+from setuptools import setup, find_packages
+
+
+PACKAGE_NAME = "azure-ai-translation-text"
+version = "1.0.0b1"
+
+with open('README.md', encoding='utf-8') as f:
+    readme = f.read()
+with open('CHANGELOG.md', encoding='utf-8') as f:
+    changelog = f.read()
+
+setup(
+    name=PACKAGE_NAME,
+    version=version,
+    description="azure-ai-translation-text",
```

Comment:
Description should be something like 'Azure Text Translation Client Library for Python'

Reply:
Done